modules/nodes-clusters-cgroups-2.adoc

```diff
@@ -0,0 +1,194 @@
+
+ifeval::["{context}" == "nodes-cluster-cgroups-2"]
+:nodes:
+endif::[]
+ifeval::["{context}" == "post-install-cluster-tasks"]
+:post:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="nodes-clusters-cgroups-2_{context}"]
+= Configuring Linux cgroup v2
+
+ifdef::post[]
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) in your cluster by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroups version 1 controllers and hierarchies in your cluster. cgroup v1 is enabled by default.
+
+cgroup v2 is the next version of the Linux cgroup API. cgroup v2 offers several improvements over cgroup v1, including a unified hierarchy, safer sub-tree delegation, new features such as link:https://www.kernel.org/doc/html/latest/accounting/psi.html[Pressure Stall Information], and enhanced resource management and isolation. 
+
+[IMPORTANT]
+====
+{product-title} cgroups version 2 support is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::post[]
+
+ifdef::nodes[]
+You enable cgroup v2 by editing the `node.config` object.
+endif::nodes[]
+
+.Prerequisites
+* You have a running {product-title} cluster that uses version 4.12 or later.
+* You are logged in to the cluster as a user with administrative privileges.
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates.
+
+.Procedure
+
+. Enable cgroup v2 on nodes:
+
+.. Edit the `node.config` object:
++
+[source,terminal]
+----
+$ oc edit nodes.config/cluster
+----
+
+.. Add `spec.cgroupMode: "v2"`:
++
+.Example `node.config` object
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Node
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2022-07-08T16:02:51Z"
+  generation: 1
+  name: cluster
+  ownerReferences:
+  - apiVersion: config.openshift.io/v1
+    kind: ClusterVersion
+    name: version
+    uid: 36282574-bf9f-409e-a6cd-3032939293eb
+  resourceVersion: "1865"
+  uid: 0c0f7a4c-4307-4187-b591-6155695ac85b
+spec:
+  cgroupMode: "v2" <1>
+...
+----
+<1> Enables cgroup v2.
+
+
+.Verification
+
+. Check the machine configs to see that the new ones were added:
++
+[source,terminal]
+----
+$ oc get MachineConfig
```

Comment:
Noticed it's `MachineConfig` in the command here, then `mc` in the next step's command. Not a big deal, but maybe sync up on one?